src/simulate/send_to_udp/HttpHandler.py

```diff
@@ -1,34 +1,37 @@
 import logging
 import requests
 import utils
+import time
 
 
 class HttpHandler:
-    def __init__(self, config_data):
+    def __init__(self, config_data, session):
         self.config_data = config_data
+        self.session = session
 
     def make_api_call(self, data):
         response = None
         try:
-            http = self.config_data[utils.PROPS_ENDPOINT]
+            endpoint = self.config_data[utils.PROPS_ENDPOINT]
         except KeyError:
             logging.error('configuration yaml is missing the\"' + utils.PROPS_ENDPOINT + '\"key')
             return
-        if http is None or http['url'] is None:
-            logging.error('End Point URL information not available to the configuration yml file')
+        if endpoint is None or endpoint['url'] is None or endpoint['token'] is None:
+            logging.error('End Point information not available to the configuration yml file')
```

Comment:
Should this return or throw an error?

Reply:
Currently it is returning None and the calling function nows how to handle this. It could also throw error.